examples/micro-speech/micro_speech.py

```diff
@@ -0,0 +1,82 @@
+# %% [markdown]
+"""
+![Renode](https://antmicro.com/OpenSource/assets/images/projects/renode.png)
+"""
+
+# %% [markdown]
+"""
+## Install requirements
+"""
+# %%
+!pip install -q ffmpeg-python pyaudioconvert git+https://github.com/antmicro/pyrenode.git git+https://github.com/antmicro/renode-colab-tools.git
+!apt install xxd --quiet
+!mkdir -p renode && cd renode && wget https://dl.antmicro.com/projects/renode/builds/renode-latest.linux-portable.tar.gz && tar -xzf renode-latest.linux-portable.tar.gz --strip 1
+!pip install -q -r renode/tests/requirements.txt
+!wget -O binary_yes https://dl.antmicro.com/projects/renode/audio_yes_1s.s16le.pcm-s_32000-b69f5518615516f80ae0082fe9b5a5d29ffebce8
+!wget -O binary_no https://dl.antmicro.com/projects/renode/audio_no_1s.s16le.pcm-s_32000-f36b60d425c9d5414a38658dbd096313d82d28c5
+
+import os
+from renode_colab_tools import *
+os.environ['PATH'] = os.getcwd()+"/renode:"+os.environ['PATH']
+
+# %%
+!mkdir -p binaries/micro_speech && cd binaries/micro_speech && wget https://github.com/antmicro/tensorflow-arduino-examples-binaries/raw/master/micro_speech/micro_speech.ino.elf # fetch prebuilt binaries
+# %% [markdown]
+"""
+## Get audio
+To change the selected option, rerun the cell
```

Comment:
is this really necessary to rerun? It's a nuisance and doesn't seem to be super needed from the algorithmic perspective

Reply:
Yes, if we want to change option or record another audio, rerun is needed to replace binary audio file.

Reply:
But why can't we solve it with a button click? Can't we have multiple actions handled?

Reply:
Yes, you are right... we don't have to rerun cell to change the option

Reply:
Yes, you are right... it can be done